src/jit/ryujit_x86/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+project(ryujit)
```

Comment:
This project name is used by jit/ryujit/standalone. It would be nice to use different name here.